code/modules/paperwork/records/info/zayin.dm

```diff
@@ -70,7 +71,7 @@
 	abno_code = "D-01-145"
 	abno_info = list(
 		"When work with T-01-145 was completed, it started to speak.",
-		"If the agent working with T-01-145 listened to it's entire story, they recieved a buff for the corresponding attribute lasting 5 minutes.")
+		"All agents present on the room while T-01-145 told it's entire story recieved a buff for the corresponding attribute, lasting 5 minutes.")
```

Comment:
No longer accurate and needs to be reverted.

Reply:
Reverted, instead corrected a spelling mistake